iothub/device/src/Transport/AmqpIot/AmqpIotExceptionAdapter.cs

```diff
@@ -40,9 +40,7 @@ internal static Exception ConvertToIotHubException(Exception exception)
         internal static Exception ConvertToIotHubException(Exception exception, AmqpObject source)
         {
             Exception ex = ConvertToIotHubException(exception);
```

Comment:
What is this doing? Did it convert it?

Reply:
I just left this method as-is. With another glance, it can be improved though.